draft-thomson-webpush-protocol.xml

```diff
@@ -285,11 +290,17 @@
               server <xref target="send"> sends push messages</xref> using this
               resource.
             </t>
+            <t hangText="receipt:">
+              A link relation of type "urn:ietf:params:push:receipt" refers to a
+              resource with read access to delivery receipts.  An application
+              server <xref target="request_receipt"> receives delivery
+              confirmations </xref> for push messages using this resource.
+            </t>
           </list>
         </t>
       </section>
     </section>
-    <section anchor="subscribe" title="Subscribing">
+    <section anchor="message_subscription" title="Subscribing to Messages">
```

Comment:
Nit: "Subscribing for Push Messages"

I try to use "push message" unless I have already used it in the same sentence.